### PR TITLE
[IMPROVEMENT] Added Emoji Category header in EmojiPicker

### LINKED ISSUE #1945: EmojiPicker missing Header for Each category
- Your Rocket.Chat Experimental app version: 4.6.0.99999999
- Your Rocket.Chat server version: 3.1.0-develop
- Device (or Simulator) you're running with: Redmi 5

In our Rocket.Chat web application the EmojiPicker has a header text for each category, this lets the user know what is that particular section of emojis are about. This header text is missing in our Rochet.Chat.ReactNative mobile client app.

### PR DESCRIPTION
**Pull request by sarthakpranesh**
@RocketChat/ReactNative

Closes #1945 

Each Category now displays a relevant header text to help the user understand what sort of emojis are present on that page. Currently, the EmojiPicker on the mobile client is very different from that of the web client. One of the features that the web client EmojiPicker has is that each category of the emojis present has a header text denoting the type of emojis under that section. This PR brings this header to the mobile client :smile: 

In Dark Mode:
ezgif com-video-to-gif

In Light Mode: 
ezgif com-video-to-gif(1)

### CODE REVIEW

**Comment by sarthakpranesh** [comment]
@diegolmello Added the required info and gifs on description.

**Comment by AgentTNT** [comment]
@sarthakpranesh - Perhaps it would be best if the headers for each list entry were that of a i18n value instead of being hardcoded to English

**Comment by sarthakpranesh** [comment]
@AgentTNT I have added the header of each emoji list to i18n. Thanks for the suggestion :+1:

File: app/containers/EmojiPicker/categories.js [hunk 1]
```diff
@@ -1,54 +1,56 @@
-const list = ['frequently', 'custom', 'people', 'nature', 'food', 'activity', 'travel', 'objects', 'symbols', 'flags'];
+import I18n from '../../i18n';
+
+const list = ['frequentlyUsed', 'custom', 'people', 'nature', 'food', 'activity', 'travel', 'objects', 'symbols', 'flags'];
 const tabs = [
 	{
 		tabLabel: '🕒',
 		category: list[0],
-		name: 'Frequently Used'
+		name: I18n.t('Frequently_Used')
```
Comment (diegolmello) [inline app/containers/EmojiPicker/categories.js:8]: `tabs` constant is created when the bundle is instantiated.
So it's never going to follow I18n changes.
You can do the test:
- Open emoji picker
- Check the label
- Change language
- Check again (it'll never change)

I know you added on `en` only, but I'm pretty sure that's what's going to happen :)

Comment (sarthakpranesh) [reply app/containers/EmojiPicker/categories.js:8]: Oh damn, I'll sure do the test and if required revert the those changes back 😊

Comment (sarthakpranesh) [reply app/containers/EmojiPicker/categories.js]: Yup, moved the i18n translations to the EmojiCategory fixes this issue :+1:

**Pull request closed without merging**